TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -218,9 +227,12 @@ protected override InitializeResult OnInitialize(InitializeParams parameters)
             var rootDirectory = new DirectoryInfo(parameters.rootPath);
             string workspaceName = rootDirectory.Name + "#" + parameters.processId;
 
-            // Initialize the workspace.
+            // Initialize the workspace.            
+#if EUROINFO_RULES
+            this.Workspace = new Workspace(rootDirectory.FullName, workspaceName, _messagesActionsQueue, Logger, CpyCopyNamesMap);
```

Comment:
Why a new constructor instead of setting a property just like the other options ?

Reply:
To me it should be done here if it come from server option, because it can come from server options or from Client configuration changed option.

Reply:
done in e4234a0